StreamChatUITestsAppUITests/Tests/StreamTestCase.swift

```diff
@@ -28,6 +29,9 @@ class StreamTestCase: XCTestCase {
     }
 
     override func tearDownWithError() throws {
+        // Reset server response delay
+        StreamMockServer.httpResponseDelay = 0.0
```

Comment:
Just thinking loudly: is there any chance that the test will fail while in the Swifter thread and `StreamMockServer.httpResponseDelay = 0.0` won't be triggered? Might be safer to put it in the `setUpWithError()`?

Reply:
done, moved to configure method